src/components/Composer.vue

```diff
@@ -379,14 +379,15 @@
 							{{ t('mail', 'Custom date and time') }}
 						</ActionRadio>
 						<ActionInput v-model="selectedDate"
+							:is-native-picker="true"
+							:min="dateToday"
 							type="datetime-local"
 							:first-day-of-week="firstDayDatetimePicker"
 							:use12h="showAmPm"
 							:formatter="formatter"
 							:format="'YYYY-MM-DD HH:mm'"
 							icon=""
 							:minute-step="5"
-							:show-second="false"
 							:disabled-date="disabledDatetimepickerDate"
```

Comment:
i dont know what this does and if we need it

Reply:
I think you can drop disabledDatetimepickerDate and disabledDatetimepickerTime

Reply:
ah, ok, then its not working so dropping it off